### PR TITLE
Minor code improvement in build definition

### PR DESCRIPTION
**Pull request by rituraj2342**

### CODE REVIEW

File: project/GenerateAnyVals.scala [hunk 1]
```diff
@@ -1,5 +1,7 @@
 package scala.build
 
+import scala.language.postfixOps
```
Comment (som-snytt) [inline project/GenerateAnyVals.scala]: postfixOps is deprecated

Comment (rituraj2342) [reply project/GenerateAnyVals.scala]: Oh. Let me revert it.

File: project/GenerateAnyVals.scala [hunk 2]
```diff
@@ -164,8 +166,8 @@ import scala.language.implicitConversions"""
       val xs1 = List(mkCoercions, mkUnaryOps, mkStringOps, mkShiftOps) map (xs => if (xs.isEmpty) xs else xs :+ "")
       val xs2 = List(
         mkBinOpsGroup(comparisonOps, numeric, _ => Z),
-        mkBinOpsGroup(bitwiseOps, cardinal, this opType _),
-        mkBinOpsGroup(otherOps, numeric, this opType _)
+        mkBinOpsGroup(bitwiseOps, cardinal, this opType),
+        mkBinOpsGroup(otherOps, numeric, this opType)
```
Comment (som-snytt) [inline project/GenerateAnyVals.scala]: not going to look at why `opType` doesn't suffice, but saying `this f arg` was a style quirk.

Comment (rituraj2342) [reply project/GenerateAnyVals.scala]: Reverted.

**Comment by rituraj2342** [comment]
@som-snytt @SethTisue All checks are passed. You can merge this now.

**Comment by SethTisue** [comment]
@rituraj2342 we don't use squash-merge in this repo, preferring to have merge commits. can you squash and force-push this into a single commit yourself, please?

**Pull request closed without merging**